clirr-ignored-differences.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- see http://www.mojohaus.org/clirr-maven-plugin/examples/ignored-differences.html -->
+<differences>
+    <difference>
+        <differenceType>7012</differenceType>
+        <className>com/google/cloud/bigtable/beam/hbasesnapshots/ImportJobFromHbaseSnapshot</className>
```

Comment:
should this be com/google/cloud/bigtable/beam/hbasesnapshots/ImportJobFromHbaseSnapsho/ImportOptions?